mobify/pinny
# Fix 73: Perform scroll action in addition to binding

Fix for Issue 73
  
**Status**: *Opened for visibility*
 
**Reviewers**: @Helen-Mobify @scalvert 
**Linked Issue**: https://github.com/mobify/pinny/issues/73
  
## Changes
- Inside `_handleKeyboardShown`, perform scroll action in addition to binding action to focus event. This seems to have fixed Issue 73
  
## How to test-drive this PR
- Pull down fix branch
- Run `grunt serve`
- Navigate to examples page (`localhost:3000/examples/` by default)
- Run steps from Issue 73 to verify fix
